.github/workflows/test.yaml

```diff
@@ -22,18 +28,21 @@ jobs:
         # When updating the minimum Python version here, also update the
         # `python_requires` from `setup.cfg`.
         # Run on latest minor release of each major python version.
-        python-version: ["3.6.7", 3.8, 3.9, pypy-3.6, pypy-3.7]
+        python-version: [3.6.7, 3.7, 3.8, 3.9, 3.10.0-alpha.5, pypy-3.6, pypy-3.7]
```

Comment:
This probably ought to be a separate PR, but since I already mentioned it in IRC I ought to go ahead and put it here too...  Anyways, with the incantation below you can avoid having to update Python versions from alpha through final release.

https://github.com/pytest-dev/pytest-twisted/blob/5b1fa21b5d92ce1b32310dd0ad2010e0232be017/.github/workflows/ci.yml#L149
```yaml
        uses: actions/setup-python@v2
        with:
          python-version: '${{ matrix.python.action }}.0-alpha - ${{ matrix.python.action }}.X'
```

Reply:
@altendky I am happy with your suggest and I think that we should only pin the minimum python micro version that we support. For all other version we should get automatic updates.

Please check with @graingert to see if we can get consensus ...but I guess that a new PR would provide the right place to discuss this and take a decision.

--------

I guess that is a question between CI stability and real life usage.

I expect that most people will use in real life the latest python micro release. 
We want to run our tests in a CI environment as close as possible to that one.

Now, automatic updates might break the CI ... but then we will have a CI that is solid green, yet Twisted fails in real life :)

Reply:
For completeness, here's an updated incantation that accounts for that incantation not working with PyPy.  Yeah, since we're calling out a minimum patch version, that'd be an extra special bit to deal with.  Aside from that, all this does is save you from having to bump from alpha to beta to rc to release.

https://github.com/pytest-dev/pytest-twisted/pull/135/files
```yaml
          # This allows the matrix to specify just the major.minor version while still
          # expanding it to get the latest patch version including alpha releases.
          # This avoids the need to update for each new alpha, beta, release candidate,
          # and then finally an actual release version.  actions/setup-python doesn't
          # support this for PyPy presently so we get no help there.
          # 
          # CPython -> 3.9.0-alpha - 3.9.X
          # PyPy    -> pypy-3.7
          python-version: ${{ fromJSON(format('["{0}", "{1}"]', format('{0}.0-alpha - {0}.X', matrix.python.action), matrix.python.action))[startsWith(matrix.python.action, 'pypy')] }}
```